### PR TITLE
Update: Add fili consumer to keep filter in sync with dateTime column

### LINKED ISSUE #1000: Add Fili Consumer
- Keep filter grain and column sync

### PR DESCRIPTION
**Pull request by kevinhinterlong**
Resolves #1000 

## Description
With the new report view it's challenging to change the parameters of a filter and would be extra work to change a parameter in both the column and filter

## Proposed Changes
- Adds a fili specific consumer which will forward updates to the dateTime column to the dateTime filter so the grains are always in sync

## License

I confirm that this contribution is made under an MIT license and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: packages/reports/addon/consumers/request/fili.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+  /**
+   * Filters actions to only those updating fili requests
+   */
+  shouldConsumeAction(_actionName: string, route: Route) {
+    const { routeName } = route;
+    const { request } = route.modelFor(routeName) as ReportModel;
+    const { type } = getDataSource(request.dataSource);
+
+    return type === 'bard';
```
Comment (jkusa) [inline packages/reports/addon/consumers/request/fili.ts:24]: 👍

File: packages/reports/tests/unit/consumers/request/fili-test.ts [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+    consumer.send(
+      RequestActions.UPDATE_COLUMN_FRAGMENT_WITH_PARAMS,
+      { modelFor },
+      request.timeGrainColumn,
+      'grain',
+      'week'
+    );
+    assert.deepEqual(dispatchedActions, [], 'no actions are called when the request is for an elide dataSource');
+  });
+});
```
Comment (jkusa) [inline packages/reports/tests/unit/consumers/request/fili-test.ts:159]: Do we already have acceptance tests that verify this behavior?

Comment (kevinhinterlong) [reply packages/reports/tests/unit/consumers/request/fili-test.ts:159]: I don't think we load any elide data sources into the reports package

Comment (jkusa) [reply packages/reports/tests/unit/consumers/request/fili-test.ts:159]: Should we update this test name to include `bard/fili`

https://github.com/yahoo/navi/blob/59189827f8c06a72166df1c1b38671b03d7fbc09/packages/reports/tests/acceptance/date-filter-test.js#L70

**Review by jkusa — APPROVED** [review]
Just the one comment, othewise 👍

**Review by jkusa — APPROVED** [review]

**Pull request merged**